src/main/java/com/iota/iri/pluggables/utxo/TransactionData.java

```diff
@@ -405,19 +407,38 @@ private String generateHash(String txnStr) {
     }
 
     public long getBalance(String account) {
+        LocalInMemoryGraphProvider provider = (LocalInMemoryGraphProvider)tangle.getPersistenceProvider("LOCAL_GRAPH");
```

Comment:
in TransactionDataTest, you need to initialize tangle, or else the test's gonna fail.